src/ducks/sharing/ShareModal.jsx

```diff
@@ -1,232 +1,42 @@
 import styles from './share.styl'
 
 import React, { Component } from 'react'
-import CopyToClipboard from 'react-copy-to-clipboard'
 import Modal from 'cozy-ui/react/Modal'
-import Toggle from 'cozy-ui/react/Toggle'
-import classnames from 'classnames'
 import { Tab, Tabs, TabList, TabPanels, TabPanel } from 'cozy-ui/react/Tabs'
-import Alerter from '../../components/Alerter'
-import ShareAutocomplete from './ShareAutocomplete'
 
-import WhoHasAccess from './WhoHasAccess'
-import { findPermSetByLink, createPermSet, deletePermSet, getShareLink, share } from '.'
+import ShareByLink from './components/ShareByLink'
+import ShareByEmail from './components/ShareByEmail'
 
 export class ShareModal extends Component {
-  constructor (props) {
-    super(props)
-    this.state = {
-      loading: true,
-      creating: false,
-      byLinkPermissions: null,
-      active: false,
-      copied: false
-    }
-  }
-
-  componentDidMount () {
-    const { _id, _type } = this.props.document
-    findPermSetByLink(_id, _type)
-      .then(byLinkPermissions => {
-        if (byLinkPermissions !== undefined) {
-          this.setState({ loading: false, byLinkPermissions, active: true })
-        } else {
-          this.setState({ loading: false })
-        }
-      })
-      .catch(() => this.onError())
-  }
-
-  toggleShareLink (active) {
-    const { _id, _type } = this.props.document
-    if (active) {
-      this.setState({ creating: true })
-      createPermSet(_id, _type)
-        .then(byLinkPermissions => this.setState({ active, byLinkPermissions, creating: false }))
-        .catch(() => this.onError())
-    } else {
-      const setId = this.state.byLinkPermissions._id
-      this.setState({ active, byLinkPermissions: null })
-      deletePermSet(setId)
-        .catch(() => this.onError())
-    }
-  }
-
-  sendSharingLinks (email, url) {
-    return share(this.props.document, email, url)
-    .then(sharing => {
-      Alerter.info('Albums.share.shareByUrl.success', { email })
-    })
-    .catch(err => {
-      Alerter.error('Error.generic')
-      throw err
-    })
-  }
-
-  onError () {
-    Alerter.error('Error.generic')
-    this.props.onClose()
-  }
-
   render () {
     const { t } = this.context
     const { onClose } = this.props
-    const { loading, active, creating, byLinkPermissions, copied } = this.state
     return (
       <Modal
         title={t('Albums.share.title')}
         secondaryAction={onClose}
       >
-        <Tabs initialActiveTab='link'>
+        <Tabs initialActiveTab='email'>
```

Comment:
I think you accidentally left that in? Or if email sharing becomes the default then I think that tab should be first in the list.